src/settings/watchers/ThemeWatcher.ts

```diff
@@ -103,7 +103,7 @@ export default class ThemeWatcher {
         // then use that over anything else. We pick the lowest possible level for the setting
         // to ensure the ordering otherwise works.
         const systemThemeExplicit = SettingsStore.getValueAt(
-            SettingLevel.DEVICE, "use_system_theme", null, false, true);
+            SettingLevel.DEVICE, "use_system_theme", null);
```

Comment:
I would probably prefer extract out another user setting (themeFallsOnConfig or something, and use `SettingsStore.getValue(UIFeature.themeFallsOnConfig) || SettingsStore.getValue(UIFeature.ChangeTheme)`) because I think there's a use-case for setting a default theme and then allowing the user to override it, that is distinct and different from the case of when the theme is set and the user can't change it.

The original change allowed this use case by default; but ultimately I think it's a relatively unimportant point, so I'm happy with the suggested code change.

Reply:
Hi Jasen, I had same suggestion in our chat for Joseph to leave default changes and add extra if needed than removing booleans. I am glad you have chosen that approach as well.

Reply:
@PhonicCanine , yeah, when we add new things we consider "minimise changes", "minimise code merge conflict", then think of other aspects they dont negatively impact these two.